lib/Page.js

```diff
@@ -98,6 +98,7 @@ class Page extends EventEmitter {
   }
 
   _onTargetCrashed() {
+    this._closed();
```

Comment:
there should be no need to emit the error event now; it should be just 'crashed'

cc @JoelEinbinder